### DISCUSSION TITLE
Frontend discussion

### DISCUSSION THREAD

**Discussion by sebastienbeau** [shopinvader/odoo-shopinvader#1300] [Workshop]
Topic to discuss
- PWA
- vuejs
-  nuxt
- discuss about architecture
- decoupling component
- base template

**Comment by Mat-moran** [comment]
We are starting with `Next.js` amazing results! It would be nice to add to the list. `SSR` static site generation and `SSR` server side rendering out of the box. Also with `SWR` or `React Query` libraries you have all the fetched data updated and cached very easily in your website, it seems that you are inside Odoo.

**Comment by Mat-moran** [comment]
Another interesting option is https://remix.run/docs/en/v1 this is an amazing framework from the creators of React router among others.
